doc/api_callback.rst

```diff
@@ -80,12 +80,16 @@
     `action` value     meaning                                                                           Appearance*
     ================== ================================================================================= ===========
     PKG_CLEANUP        `package` cleanup is being performed.                                             3
-    PKG_DOWNGRADE      `package` is being downgraded.                                                    2
+    PKG_DOWNGRADE      new `package` is being installed as a downgrade                                   2
```

Comment:
Missing a dot at the end on this line.

I'm also not sure what the _new_ word should mean here. Is it really a new package if it's a downgrade? I may not have the context, but it seems weird to me.